modules/gdscript/tests/scripts/Traits/analyzer/errors/class_extends_trait.out

```diff
@@ -0,0 +1,2 @@
+GDTEST_ANALYZER_ERROR
+>> ERROR at line 1: Could not inheritance from trait, "TraitA".
```

Comment:
GDScript analyzer and runtime tests now support tracking multiple errors in a single file. Please group similar errors into single files instead of creating many small files. Same goes for feature tests. Yes, you shouldn't put everything in one file, but I think there are too many of them now.